### PR TITLE
Sprint4 Create UI History Detail

### PR DESCRIPTION
**Pull request by nhahv0902**

### CODE REVIEW

File: app/src/main/java/com/framgia/fdms/data/model/HistoryDetail.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    @Bindable
+    public String getUserAssigner() {
+        return mUserAssigner;
+    }
+
+    public void setUserAssigner(String userAssigner) {
+        mUserAssigner = userAssigner;
+        notifyPropertyChanged(BR.userAssigner);
+    }
+
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/data/model/HistoryDetail.java]: remove bl

File: app/build.gradle [hunk 2]
```diff
@@ -50,4 +50,6 @@ dependencies {
 
   compile 'com.github.PhilJay:MPAndroidChart:v3.0.2'
   testCompile 'junit:junit:4.12'
+
```
Comment (DoanVanToan) [inline app/build.gradle]: remove bl

File: app/src/main/java/com/framgia/fdms/screen/devicedetail/DeviceDetailPagerAdapter.java [hunk 3]
```diff
@@ -35,7 +35,7 @@ public Fragment getItem(int position) {
             case DEVICE_INFOMATION:
                 return DeviceUsingHistoryFragment.newInstance();
             case DEVICE_HISTORY:
-                return DeviceUsingHistoryFragment.newInstance();
+                return HistoryDetailFragment.newInstance();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/devicedetail/DeviceDetailPagerAdapter.java]: HistoryDetailFragment > DeviceDetailHistoryFragment

File: app/src/main/res/layout/item_history_detail.xml [hunk 4]
```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    >
+    <data>
+        <variable
+            name="historyDetail"
+            type="com.framgia.fdms.data.model.HistoryDetail"
+            />
+        <variable
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_history_detail.xml]: Khai báo viewModel làm gì?

Comment (nhahv0902) [inline app/src/main/res/layout/item_history_detail.xml]: fixed

**Pull request merged**